RTQA/iPython/extensions/hybrid_tags.py

```diff
@@ -0,0 +1,349 @@
+"""Contains class and functions for Praxi algorithm"""
```

Comment:
This file was contained in Brian's fork of ai-for-cloud.ops. I'm not quite sure what this file does, neither do I know who exactly authored it. 

Changes made to this file are at line 25, 158-159 and 286.
At line 25, I added the hard-coded path to the Vowpal Wabbit binary file on my computer. I don't know any way to calculate the binary's path programmatically.
At line 158 and 286, the arguements passed to the vw binary are processed through repr() to avoid problems with escape characters on different OSs.